### DISCUSSION TITLE
ERD diagram

### DISCUSSION THREAD

**Discussion by windmillcode0** [mtxr/vscode-sqltools#1317] [Ideas]
Can we get ERD diagram support use this ticket for reference
https://github.com/mtxr/vscode-sqltools/issues/1304

**Comment by windmillcode0** [comment]
If we can at least get the instruction on how to install the driver along with the main app in extension development that would really work

**Reply by gjsjohnmurray** [reply]
> how to install the driver

I'm unclear which driver you mean.

Instructions (though somewhat terse) about building from source are at https://vscode-sqltools.mteixeira.dev/en/contributing/building

On another thread I wrote more extensive steps for using a Docker container for development:

https://github.com/mtxr/vscode-sqltools/discussions/1213#discussioncomment-7557873

This repo includes the source for the 4 "official" drivers, and your VS Code workspace will set up debug configurations suitable for debugging inside these driver extensions and/or inside the main extension.

Sometime I should write notes about how I do that debugging. Maybe warrants a fresh thread here.

**Reply by gjsjohnmurray** [reply]
#1214 is about debugging.
